CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 0.8.0
+
+* Fix HTMPredictionModel prediction using SDRClassifier (#3665)
+* Allow multiple classifications for each record to SDRClassifier (#3669)
+* Remove CLAClassifier (#3665)
+* Add capnp serialization to TMRegion (#3657)
+* Document ExperimentDescriptionAPI (#3679)
+
```

Comment:
Interestingly, I don't see the merge commit for #3667 when I run this command:

```
git log --merges 0.7.0..HEAD
```

Are you using one of the other merge methods that doesn't create a merge commit?